sdk/storage/azure-storage-blob/build.gradle

```diff
@@ -17,7 +17,6 @@ android {
 
     sourceSets {
         androidTest.assets.srcDirs += files("$projectDir/schema".toString())
-        test.java.srcDirs += files("${project(':sdk:core:azure-core').projectDir}/src/test/java/com/azure/android/core/common")
```

Comment:
Did we figure out a way to share test classes from Azure Core?

Reply:
Yep. I copied them over. This works, but if you re-import the project into Android Studio it no longer understands the project structure of Core because it mistakenly thinks that the src directory belongs to azure-storage-blob. Building from the commandline still works fine obviously, it's just a mistake in how Studio infers the project structure from the gradle files. I wasn't able to correct it either, "modules" section of the project structure dialog didn't show any of the libraries any more. After a couple hours knocking my head against the desk I figured out this line was causing the issue.